parcels/particleset.py

```diff
@@ -496,16 +496,20 @@ def density(self, field=None, particle_val=None, relative=False, area_scale=Fals
 
         :param field: Optional :mod:`parcels.field.Field` object to calculate the histogram
                       on. Default is `fieldset.U`
-        :param particle_val: Optional numpy-array of values to weigh each particle with.
-                             Default is 1 for each particle
+        :param particle_val: Optional numpy-array of values to weigh each particle with,
+                             or string name of particle attribute to use weigh particles with.
```

Comment:
In Parcels, a particle attribute is normally called a `Variable`. Use that word here, too?